### PR TITLE
Getting started with recoil in react

### PR DESCRIPTION
**Pull request by Creeland**

### CODE REVIEW

File: getting-started-with-recoil-in-react-1fca/notes/01_react-create-shared-state-with-recoil-in-a-react-app.md.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<TimeStamp start="2:20" end="2:27">
+  
+  Atom keys must be unique
+  
+</TimeStamp>
+
+<TimeStamp start="2:46" end="2:54">
+  
+  useRecoilState is required to use atoms. [recoiljs.org - useRecoilState](https://recoiljs.org/docs/api-reference/core/useRecoilState/)
```
Comment (lsminter) [inline getting-started-with-recoil-in-react-1fca/notes/01_react-create-shared-state-with-recoil-in-a-react-app.md.md:36]: I think having a definition here along with the link to the docs is a good way to go. Save the users from having to click on the docs if they don't want to.

File: getting-started-with-recoil-in-react-1fca/notes/02_react-read-a-value-from-recoil-with-userecoilvalue.md.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<TimeStamp start="1:09" end="1:19">
+  
+  `useRecoilValue` is used to read state
```
Comment (lsminter) [inline getting-started-with-recoil-in-react-1fca/notes/02_react-read-a-value-from-recoil-with-userecoilvalue.md.md:3]: A link to the docs like you did in the previous lesson with useRecoilState would be good to have. Keeps things consistent and gives the user a way to get more info if they want.

File: getting-started-with-recoil-in-react-1fca/notes/05_react-use-async-functions-in-a-recoil-selector.md.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+<TimeStamp start="2:09" end="2:16">
+  
+  You can use `useRecoilValue` the exact same way you would use it with synchronous selectors
+  
+</TimeStamp>
+
+<TimeStamp start="2:30" end="2:40">
+  
+  [React Suspense](https://reactjs.org/docs/concurrent-mode-suspense.html)
```
Comment (lsminter) [inline getting-started-with-recoil-in-react-1fca/notes/05_react-use-async-functions-in-a-recoil-selector.md.md:15]: A short definition of Suspence would be a good idea. The instructor doesn't go into much detail about it at all. I don't think he even mentions that it's currently experimental and subject to change.

**Review by lsminter — CHANGES_REQUESTED** [review]
There is definitely some good info that we can put into notes on lesson 5. The instructor talks about "violating one of the rules of hooks by using a hook inside of something that is not calling a hook". Some more info there as to why would be really good for the learner. 

The file names are all suffixed with `.md.md`.

File: getting-started-with-recoil-in-react-1fca/notes/02_react-read-a-value-from-recoil-with-userecoilvalue.md.md (hunk 2 above)
Comment (Creeland) [reply getting-started-with-recoil-in-react-1fca/notes/02_react-read-a-value-from-recoil-with-userecoilvalue.md.md:3]:
```suggestion
  `useRecoilValue` is used to read state. [recoiljs.org - useRecoilValue](https://recoiljs.org/docs/api-reference/core/useRecoilValue/)
```

File: getting-started-with-recoil-in-react-1fca/notes/05_react-use-async-functions-in-a-recoil-selector.md.md (hunk 3 above)
Comment (Creeland) [reply getting-started-with-recoil-in-react-1fca/notes/05_react-use-async-functions-in-a-recoil-selector.md.md:15]:
```suggestion
  "Suspense for Data Fetching is a new feature that lets you also use <Suspense> to declaratively “wait” for anything else, including data." - [reactjs.org](https://reactjs.org/docs/concurrent-mode-suspense.html)
  
 React Suspense is experimental and subject to change.
```

**Pull request merged**